src/general/MicrobeActionInterferenceMode.cs

```diff
@@ -0,0 +1,29 @@
+﻿/// <summary>
+///   Describes how two microbe actions interference with each other.
+/// </summary>
+/// <remarks>
+///   <para>Used for MP calculation</para>
+/// </remarks>
+public enum MicrobeActionInterferenceMode
```

Comment:
```suggestion
public enum ActionInterferenceMode
```
I think that would be a better name as this is used by the generically named class `CombinableActionData`